Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -2616,6 +2621,62 @@ namespace OpenInfraPlatform
 				}
 #endif
 
+#if defined(OIP_MODULE_EARLYBINDING_IFC4X3_RC4)
+
+				/*! \brief Calculates a trimming point on the sine curve.
+				* \param[in] sine			        A pointer to data from \c IfcSine.
+				* \param[in] parameter				A pointer to data from \c IfcParameterValue.
+				* \return							The location of the trimming point.
+				* \note
+				*/
+				template <>
+				carve::geom::vector<3> getPointOnCurve(const EXPRESSReference<typename IfcEntityTypesT::IfcSine>& sine,
+					const typename IfcEntityTypesT::IfcParameterValue& parameter) const noexcept(false)
+				{
+					return getPointOnCurve(sine, parameter * this->UnitConvert()->getLengthInMeterFactor());
+				}
+				template <>
+				carve::geom::vector<3> getPointOnCurve(const EXPRESSReference<typename IfcEntityTypesT::IfcSine>& sine,
+					const typename IfcEntityTypesT::IfcNonNegativeLengthMeasure& parameter) const noexcept(false)
+				{
+					return getPointOnCurve(sine, parameter * this->UnitConvert()->getLengthInMeterFactor());
+				}
+				carve::geom::vector<3> getPointOnCurve(const EXPRESSReference<typename IfcEntityTypesT::IfcSine>& sine,
+					const typename IfcEntityTypesT::IfcParameterValue& parameter) const noexcept(false)

```

Comment:
This function's signature is the same as 10 lines above. Please double check.

Reply:
Resolved in [d4d80c8](https://github.com/tumcms/Open-Infra-Platform/pull/553/commits/d4d80c8a2ce0a978560901d0990d8b5543b3332e)